t/mldistwatch-misc.t

```diff
@@ -339,6 +339,35 @@ subtest "check overlong versions" => sub {
   );
 };
 
+subtest "version is not a lax version string" => sub {
+  my $pause = PAUSE::TestPAUSE->init_new;
+  $pause->upload_author_fake(MONSTER => 'Hex-Version-1.234.tar.gz', {
+    append => [
+      {
+        file => "lib/Hex/Version/NoJoke.pm",
+        content => <<'EOT',
```

Comment:
What now you're too good for indented heredocs?

Reply:
Heh.  I think I copied this from another test, and until recently we were on 5.16 or something!